lib/fauxpaas/command.rb

```diff
@@ -59,12 +59,7 @@ def action
 
     def execute
       signature = instance.signature(reference)
-
-      release = Release.new(
-        artifact: Fauxpaas.artifact_builder.build(signature),
-        deploy_config: DeployConfig.from_ref(signature.deploy, Fauxpaas.ref_repo)
-      )
-      status = release.deploy
+      status = Release.new(signature).deploy
```

Comment:
This seems to be bouncing back and forth. We should clarify exactly what we think the responsibilities for Artifact / Release are.